docs/gettingstarted/manifest-deployer-tutorial/README.md

```diff
@@ -0,0 +1,358 @@
+│   ├── my-deployment.yaml
+│   ├── my-secret.yaml
+│   └── my-target.yaml
+├── prepare-demo-environment.md
+└── README.md
+```
+
+| Demo environment | Productive environment |
+| ------------- |-------------|
+| For demo purposes we will use a simplified setup with one cluster, which contains all necessary parts. | In a productive environment could be separated. |
+| ![alt text](README-env-demo.svg "demo all-in-one cluster") | ![alt text](README-env-productice.svg "One demo cluster and other possible setup")|
+
+### Install the Landscaper together with an OCI registry
+
+The landscaper-cli provides a convenient quick start option for installing the Landscaper plus an OCI registry:
+
+```bash
+export OCI_USER=oci-user
+export OCI_PASSWD='PSST!DONOTTELL!'
+
+landscaper-cli quickstart install \
+  --kubeconfig ~/.kube/config-demo.yaml \
+  --install-oci-registry \
+  --install-registry-ingress \
+  --registry-username $OCI_USER \
+  --registry-password $OCI_PASSWD
+```
+
+A successful installation should look like this:
+
+``` text
+Landscaper installation succeeded!
+
+The OCI registry can be accessed via the URL https://o.ingress.p.democlust.shoot.k8s.myprovider.com
+It might take some minutes until the TLS certificate is created
+```
+
+Remember, you need this URL to access the OCI registry later on:
+
+``` bash
+export OCI_REGISTRY=o.ingress.p.democlust.shoot.k8s.myprovider.com
+```
+
+Check if all needed components of Landscaper is available:
+
+``` bash
+kubectl get pods -n landscaper --kubeconfig ~/.kube/config-demo.yaml
+```
+
+The installation should contain
+
+- Landscaper
+- Landscaper webhooks
+- Container deployer
+- Helm deployer
+- Manifest deployer
+- OCI Registry
+
+```text
+NAME                                                    READY   STATUS    RESTARTS   AGE
+container-default-container-deployer-5c6as449f9-dz4x6   1/1     Running   0          25s
+helm-default-helm-deployer-7cd3fd9797-5c5rk             1/1     Running   0          41s
+landscaper-654cbc4568-x6bs7                             1/1     Running   0          53s
+landscaper-webhooks-58995rff45-htwcf                    1/1     Running   0          53s
+manifest-default-manifest-deployer-7fbc87ef79-xpfnq     1/1     Running   0          31s
+oci-registry-6654f55648-wg7bq                           1/1     Running   0          60s
+
+```
+
+Check availability of OCI registry:
+
+``` bash
+curl --location --request GET https://$OCI_REGISTRY/v2/_catalog -u "$OCI_USER:$OCI_PASSWD"
+```
+
+The curl should return this:
+
+```text
+{"repositories":[]}
+```
+
+## Deploy applications via Landscaper
+
+### 1. Push demo image content into OCI registry
+
+For this tutorial an OCI image is provided. The image contains a dummy application just for demonstration purposes. It does nothing, just keeps the container running.
+
+The image is located at /manifest-deployer-tutorial/demo-content/hello.tar.
+For the next steps, dockerd must be up and running in demo environment.
```

Comment:
Who knows what dockerd is? Rename this to "For the next steps you need a running docker environment". This is also not mentioned on dev setup page.